data.sql

```diff
@@ -1,5 +1,7 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES('Agumon','03/02/2020',0,TRUE,10.23)
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES('Gabumon','15/11/2018',2,TRUE,8.0)
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES('Pikachu','07/01/2021',1,FALSE,15.04)
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES('Devimon','12/05/2017',5,TRUE,11.0)
```

Comment:
- [ ] Kindly include `;` at the end of each statement to end it, It causes errors when we do not end statements.

- [ ] I got a date format error when I tried to insert the data, Kindly consider using the date format `year/month/day` or `month/day/year` for all your data which I tried and it worked. Kindly fix the error.

Errors from the data format;
<img width="547" alt="Code_bfxQzgJSHl" src="https://user-images.githubusercontent.com/81468675/165507897-078bc49c-094f-48b4-848f-a626d9d4ba6a.png">


After I changed the date format;
<img width="531" alt="Code_uO2kYllBW2" src="https://user-images.githubusercontent.com/81468675/165507947-4342ab12-c53f-47f3-99fb-5de0609a4103.png">